packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -121,36 +127,37 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
     public async createDocumentService(
         resolvedUrl: IResolvedUrl,
         logger?: ITelemetryBaseLogger,
-        epochTracker?: EpochTracker,
+        cacheAndTrackerArg?: ICacheAndTracker,
     ): Promise<IDocumentService> {
+        const odspResolvedUrl = (resolvedUrl as IOdspResolvedUrl);
```

Comment:
can you add a type guard/assert function to ensure this is actually an IOdspResolvedUrl, seems like that would be a useful function to have in general